core/corehttp/gateway_handler.go

```diff
@@ -602,3 +623,42 @@ func getFilename(s string) string {
 	}
 	return gopath.Base(s)
 }
+
+func (i *gatewayHandler) searchUpTreeFor404(r *http.Request, parsedPath ipath.Path) (ipath.Resolved, error) {
+	filename404, err := preferred404Filename(r.Header["Accept"])
```

Comment:
```suggestion
	filename404, err := preferred404Filename(r.Header.Get("Accept"))
```

Reply:
Ah, now I see. Let's call `r.Header.Values` (go 1.14). I'll go ahead and update our CI.